.env

```diff
@@ -0,0 +1,28 @@
+
+INDABA_PG_USERNAME=amida
+INDABA_PG_TESTUSER=amida
+INDABA_PG_PASSWORD=amida
+INDABA_PG_DB=indaba
+INDABA_PG_HOSTNAME=indaba-db
+JWT_SECRET=veryverysecret
+AUTH_SALT=supersalty
+
+ENV=dev
+JWT_MODE=hmac
+PG_DB=auth
+PG_USER=amida
+PG_PASSWD=amida
+PG_HOST=postgres
+MAILER_SERVICE_PROVIDER=Gmail
+CREATE_USER_ADMIN=false
+
+SURVEY_SERVICE_DB_NAME=survey
+SURVEY_SERVICE_DB_PORT=5432
+SURVEY_SERVICE_DB_USER=amida
+SURVEY_SERVICE_DB_PASS=amida
+SURVEY_SERVICE_DB_DIALECT=postgres
+SURVEY_SERVICE_DB_POOL_MAX=5
+SURVEY_SERVICE_DB_POOL_MIN=0
+SURVEY_SERVICE_DB_POOL_IDLE=10000
+SURVEY_SERVICE_LOGGING_LEVEL=emerg
+SURVEY_SERVICE_CORS_ORIGIN=*
```

Comment:
We are missing some env vars here: this is the complete list I have on greyscale:
```
INDABA_PG_USERNAME=
INDABA_PG_TESTUSER=
INDABA_PG_PASSWORD=
INDABA_PG_DB=
INDABA_PG_HOSTNAME=
AUTH_SALT=
JWT_SECRET=
SURVEY_SERVICE_URL=
AUTH_SERVICE_URL=
DOMAIN=
GMAIL_PASS=
MESSAGE_SERVICE_URL=
SYS_MESSAGE_USER=
SYS_MESSAGE_PASSWORD=
S3_BUCKET=
AWS_ACCESS_KEY_ID=
AWS_SECRET_ACCESS_KEY=
MAILER_EMAIL_ID=
MAILER_PASSWORD=
MAILER_SERVICE_PROVIDER=
ADMIN_USER_EMAIL=
ADMIN_USER_FIRST_NAME=
ADMIN_USER_LAST_NAME=
```